### DISCUSSION TITLE
Proxy Protocol V2 Support

### DISCUSSION THREAD

**Discussion by Andrei-Paul** [itzg/docker-minecraft-server#1283] [Q&A]
My minecraft server instance (Bukkit flavor) runs in a kubernetes cluster, using an nginx tcp ingress.
Some plugins have features that rely on the player IP, however, I can't get the player IP unless I turn on Proxy Protocol support for the minecraft server ingress.
Either I don't get it (I have been dumb before), or there is no support for Proxy Protocol in this container / in the Bukkit server itself.

Is there anyway to take advantage of the Proxy Protocol using this container? Could this be somehow implemented?

**Comment by itzg** [comment]
Proxy protocol support has nothing to do with this image. It depends on the server type used.
